bot.py

```diff
@@ -23,32 +23,37 @@ async def updateSuggestions():
 async def checkSuggestions():
     await bot.wait_until_ready()
     while True:
-        if bot.suggestQueue:
-            message=bot.suggestQueue[0]
-            if (message.created_at.utcnow()-message.created_at).seconds>(6)*3600 or (message.created_at.utcnow()-message.created_at).days>0:
-                message=bot.suggestQueue.popleft()
-                approvals = get(message.reactions, emoji="✅")
-                denials = get(message.reactions, emoji="❌")
-                if approvals.count>denials.count:
+        for message in bot.suggestQueue:
+            approvalsObject=get(message.reactions, emoji="✅")
+            denialsObject=get(message.reactions, emoji="❌")
+            approvals=approvalsObject.count-(bot.user in set(await approvalsObject.users().flatten())) #gets # of yes reactions that isn't the bot
+            denials=denialsObject.count-(bot.user in set(await denialsObject.users().flatten()))     #gets # of no reactions that isn't the bot
```

Comment:
Why are you not including bots if the bot auto un-reacts? There shouldnt be a bot reaction on it ever

Reply:
in case the bot still has the reaction b/c no one has voted